apps/preview/next/pages/showcases/Breadcrumbs/BreadcrumbsWithIcon.tsx

```diff
@@ -78,14 +79,16 @@ export function Showcase() {
               >
                 <SfIconHome size="sm" />
               </SfLink>
-            ) : (
+            ) : index !== breadcrumbs.length ? (
```

Comment:
```suggestion
            ) : index < breadcrumbs.length - 1 ? (
```
probably some first implementation leftover

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2706/commits/977b5ca061ed22d14a193404170714441edfe707 thanks :)